File: dummy-ai-client/NetworkStandard/MessageContainer/message_container.py
import uuid
from NetworkStandard.MessageTypeEnum.messagetype_enum import MessageTypeEnum


class MessageContainer:

    def __init__(self, client_Id, type_enum, creation_date, debug_message):
        self.client_Id = client_Id
        self.type_enum = type_enum
        self.creation_date = creation_date
        self.debug_message = debug_message

File: frontend/core/src/com/sopra/ntts/model/NetworkStandard/MessageContainer/MessageContainer.java
package com.sopra.ntts.model.NetworkStandard.MessageContainer;

import com.sopra.ntts.model.NetworkStandard.MessageTypeEnum.MessageTypeEnum;

import java.util.Date;
import java.util.UUID;

/**
 * Basic MessageContainer for the communication between the client and server.
 *
 * @author Standardisierungsdokument & Sedat Qaja
 */
public class MessageContainer {
    public UUID clientId;
    public MessageTypeEnum type;
    public Date creationDate;
    public String debugMessage;

    public MessageContainer(UUID clientId, MessageTypeEnum type, Date creationDate, String debugMessage) {
        this.clientId = clientId;
        this.type = type;
        this.creationDate = creationDate;
        this.debugMessage = debugMessage;
    }

    @Override
    public String toString() {
        return "MessageContainer{" +
                "playerId=" + clientId +
                ", type=" + type +
                ", creationDate=" + creationDate +
                ", debugMessage='" + debugMessage + '\'' +
                '}';
    }
}
